src/app/containers/MostRead/Canonical/index.test.jsx

```diff
@@ -6,10 +6,15 @@ import {
   setStalePromoTimestamp,
   setFreshPromoTimestamp,
   renderMostReadContainer,
-} from '../utilities/testHelper';
+} from '../utilities/testHelpers';
 
 let container;
 
+// Most Read is currently turned off for these services because data is not available for them,
+// but once they are enabled, these overrides can be removed.
+newsConfig.default.mostRead.hasMostRead = true;
```

Comment:
Why are these values being set here?

Reply:
https://github.com/bbc/simorgh/pull/5486/files#diff-08275765078687909d11f4f4f471d568R165

because for the time being, News/zhongwen will be switched off

Reply:
since it's only for a brief time, I thought it would just be easier to temporarily enable them like this so that I don't have to udpate the tests too much 😅

Reply:
I think it would be best to have the correct value in config!

Reply:
Aha, ok I understand why you're doing this now after we spoke. Rather than modifying this value, could you create a new object?

Reply:
yep that sounds more sensible